server/pkg/provision/auth.go

```diff
@@ -43,3 +61,119 @@ func emailSettings(ctx context.Context, s store.Storer) error {
 		})
 	})
 }
+
+// addAuthSuperUsers assigns BYPASS roles to users from AUTH_PROVISION_SUPER_USER value
+// When in Production, Corteza should stop and report an error.
+func addAuthSuperUsers(ctx context.Context, log *zap.Logger, s store.Storer, envOpt options.EnvironmentOpt) (err error) {
+	var (
+		val, has = os.LookupEnv("AUTH_PROVISION_SUPER_USER")
+	)
+
+	if envOpt.IsProduction() {
```

Comment:
This condition will cause Corteza to error out in production regardless if the env value is set or not